File: juego/src/main/java/com/juego/juego/dto/Game.java
package com.juego.juego.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import java.util.List;

import jakarta.annotation.Nullable;

@Data
@AllArgsConstructor
public class Game {
    private String id;
    private List<Player> players;
    private Integer necesaryPlayers;
    private Boolean roundReset;
    private Boolean ended;
    private Integer round;
    private List<Card> cards;
    private Card cardBack;
    private List<Card> midCards;
    private @Nullable Card pinte;
    private @Nullable Player whoWin;
}

File: ng-juego-socket/src/app/models/game.ts
import { Card } from "./card";
import { User } from "./user";

export interface Game {
    id: string;
    players: User[];
    necesaryPlayers: number;
    roundReset: boolean;
    ended: boolean;
    round: number;
    cards: Card[];
    cardBack: Card;
    midCards: Card[];
    pinte : Card | null;
    whoWin : User | null;
}